staging/src/k8s.io/api/admissionregistration/v1beta1/types.go

```diff
@@ -49,8 +49,32 @@ type Rule struct {
 	// Depending on the enclosing object, subresources might not be allowed.
 	// Required.
 	Resources []string `json:"resources,omitempty" protobuf:"bytes,3,rep,name=resources"`
+
+	// Scope specifies the scope of this rule.
+	// Valid values are "Cluster", "Namespaced", and "*"
+	// "Cluster" means that only cluster-scoped resources will match this rule.
+	// Namespace API objects are cluster-scoped.
+	// "Namespaced" means that only namespaced resources will match this rule.
```

Comment:
I'm wondering if we should make this `Namespace`.  We call scopes `Cluster` and `Namespace` in other places (technical doc, mostly).  I guess we used `namespaced: true` in APIResource for discovery.  Shame we didn't use `scope: Namespaced` there too.

Reply:
CRD scope uses "Cluster" and "Namespaced", was intended to match that